aten/src/ATen/Declarations.cwrap

```diff
@@ -2676,7 +2676,6 @@
   types:
```

Comment:
Once https://github.com/pytorch/pytorch/pull/20622 is in, remove this whole block: 
```
[[
  name: _th_cauchy_
  types:
    - floating_point
  backends:
    - CPU
  cname: cauchy
  variants: function
  return: self
  arguments:
    - THTensor* self
    - arg: THGenerator* generator
      default: nullptr
      kwarg_only: True
    - arg: double median
      default: 0
    - arg: double sigma
      default: 1
]]
```